dizoo/mujoco/config/halfcheetah_d4pg_default_config.py

```diff
@@ -42,7 +42,7 @@
         collect=dict(
             n_sample=8,
             unroll_len=1,
-            noise_sigma=0.2,  # [0.1, 0.2]
```

Comment:
why remove these comments

Reply:
I confirmed with David this morning to make sure these are parameter suggestions rather than debugging leftover. will add this back